Getting this on darwin intel:
```
error[E0308]: mismatched types
    --> src/vsmtp/vsmtp-common/src/libc_abstraction.rs:80:52
     |
80   |     match unsafe { libc::initgroups(user.as_ptr(), gid) } {
     |                    ----------------                ^^^ expected `i32`, found `u32`
     |                    |
     |                    arguments to this function are incorrect
```

--- a/pkgs/servers/mail/vsmtp/default.nix
+++ b/pkgs/servers/mail/vsmtp/default.nix
@@ -44,6 +44,7 @@ rustPlatform.buildRustPackage rec {
     description = "A next-gen mail transfer agent (MTA) written in Rust";
     homepage = "https://viridit.com";
     license = licenses.gpl3Plus;
+    platforms = platforms.linux;
     maintainers = with maintainers; [ nickcao ];
   };
 }

